aesara/tensor/math_opt.py

```diff
@@ -2283,37 +2284,32 @@ def local_log1p(fgraph, node):
 
 
 # TODO: in canonicalize, change log10 and log2 -> log
-@register_stabilize
-@register_specialize
```

Comment:
We don't want to remove these optimizations from the "stabilization" and "specialization" databases; otherwise, we&mdash;for instance&mdash;remove the ability to apply all stabilizations/specializations with a single optimizer query, and people are forced to either use all the `fast_run` optimizations or know about this particular optimization and apply it manually.